docs/standards/smart-contracts/lsp0-erc725-account.md

```diff
@@ -22,6 +22,16 @@ This contract could be used as a _blockchain-based account_ by humans, machines,
 
 ---
 
+
+## Behavior
+
+All ownable functions such as `execute(..)`, `setData(..)`, `transferOwnership(..)`, and `renounceOwnership(..)` can be called by the owner and any address that is allowed by the owner conforming to the [LSP20-CallVerification](../universal-profile/lsp0-erc725account.md#lsp20---call-verification) behavior.
+
+If an address is calling the `execute(..)` function and is not the owner, the account contract will forward the call to the owner. The execution of the function will only continue if the owner returns a specific magic value, which indicates that the caller is allowed to execute the function. The magic value can also determine if there should be any post execution check on the owner. This same behavior applies to other ownable functions as well.
```

Comment:
> indicating that the caller can execute the function

I still prefer "indicating that the caller can execute the function" because I see a difference between "is allowed" and "can". "Can" point to the ability to do it from the caller, and "is allowed to" point that the caller is allowed and could do the tx